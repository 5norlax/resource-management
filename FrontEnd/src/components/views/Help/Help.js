import React from 'react'
import { Container } from 'reactstrap'
import Header from '../../layout/Header/Header'

export default function Help() {

  return (
    <div>
      <Header name={'Help'} />
      <Container>
        <div>
          <h2>How To:</h2>
          <p>Coming Soon...</p>
          <p>
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
          </p>
          <h2>Shortcuts:</h2>
          <p>Navigate to Overview Page: &nbsp;&nbsp; SHIFT + o</p> 
          <p>Navigate to Resource Page: &nbsp;&nbsp; SHIFT + r</p> 
          <p>Navigate to Help Page: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SHIFT + h</p> 
          <p>Navigate to Settings Page: &nbsp;&nbsp;&nbsp; SHIFT + s</p> 
        </div>
      </Container>
    </div>
  )
}