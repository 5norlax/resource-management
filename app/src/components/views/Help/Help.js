import React from 'react'
import { Container } from 'reactstrap'
import Header from '../../layout/Header/Header'

export default function Help() {
  return (
    <div>
      <Header name={'Help'} />
      <Container>
        <div>
          <h2>Shortcuts:</h2>
          <p>Navigate to Resource Page: &nbsp;&nbsp; SHIFT + r</p>
          <p>
            Navigate to Help Page:
            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SHIFT + h
          </p>
          <p>Navigate to Settings Page: &nbsp;&nbsp;&nbsp; SHIFT + s</p>
        </div>
      </Container>
    </div>
  )
}
