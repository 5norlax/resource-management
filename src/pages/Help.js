import React from 'react';
import { Container, Button } from 'reactstrap';
import { Link } from 'react-router-dom';
import Header from '../components/Header';

const Help = () => {

    return (
        <div>
            <Header name={'Help'} />
            <Container>
                <div>
                    <p>
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                    </p>
                </div>
                <Header name={'Additional Links'} />
                <Link to="/addresource" className="col-xl-2 col-lg-2 col-md-3 col-sm-3 col-xs-3">
                    <Button type="button" color="primary">Add Resource Form</Button>
                </Link>
                <Link to="/profile" className="col-xl-2 col-lg-2 col-md-3 col-sm-3 col-xs-3">
                    <Button type="button" color="primary">Proifle Page</Button>
                </Link>
                <Link to="/addjob" className="col-xl-2 col-lg-2 col-md-3 col-sm-3 col-xs-3">
                    <Button type="button" color="primary">Add Job Page</Button>
                </Link>
            </Container>
        </div>
    );
}

export default Help;