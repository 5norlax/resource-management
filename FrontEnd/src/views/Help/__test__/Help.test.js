import React from 'react';
import { render, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Help from '../Help';

afterEach(cleanup);

describe('Test Help', () => {

  it('matches snapshot', () => {
    const tree = renderer.create(<Help />).toJSON()
    expect(tree).toMatchSnapshot();
  });

});