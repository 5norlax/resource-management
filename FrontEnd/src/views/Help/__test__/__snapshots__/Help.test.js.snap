// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Help matches snapshot 1`] = `
<div>
  <div>
    <hr />
    <h1
      className="text-center"
      data-testid="header"
    >
      Help
    </h1>
    <hr />
  </div>
  <div
    className="container"
  >
    <div>
      <h2>
        How To:
      </h2>
      <p>
        Coming Soon...
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </p>
      <h2>
        Shortcuts:
      </h2>
      <p>
        Navigate to Overview Page:    SHIFT + o
      </p>
      <p>
        Navigate to Resource Page:    SHIFT + r
      </p>
      <p>
        Navigate to Help Page:           SHIFT + h
      </p>
      <p>
        Navigate to Settings Page:     SHIFT + s
      </p>
    </div>
  </div>
</div>
`;
